### PR TITLE
Add Spandan

### PR DESCRIPTION
**Pull request by Spandamn**

### CODE REVIEW

File: data/mods/ssb/moves.ts [hunk 1]
```diff
@@ -3456,6 +3456,37 @@ export const Moves: {[k: string]: ModdedMoveData & {gen?: number}} = {
...
+		flags: {contact: 1, protect: 1, mirror: 1},
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Sludge Bomb', target);
+			this.add('-anim', source, 'Sludge Wave', target);
+		},
+		onEffectiveness(typeMod, target, type) {
+			return 0;
```
Comment (xInstruct) [inline data/mods/ssb/moves.ts:3480]: This move will always be neutral effective, are you sure about this?

**Comment by ghost** [comment]
Everything else looks fine

**Comment by Spandamn** [comment]
Done @xInstruct

**Comment by ghost** [comment]
looks good

**Pull request merged**